apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -445,12 +445,12 @@ def evaluate_with_source(
 
 def convert_string_to_sympy(
     expr: str,
-    variables: None | list[str] = None,
+    variables: None | list[str] | tuple[str, ...] = None,
```

Comment:
Huh, interesting. I had hoped Python would have some kind of type for "sequence of X" but it seems like this is the best option we have.

Reply:
Do you want to use an `Iterable[str]` instead?

Reply:
Ah! Yeah, I think that's exactly what we want.

Reply:
Resolved in [e3e22cf](https://github.com/PrairieLearn/PrairieLearn/pull/11111/commits/e3e22cfb7e63dec401450528430af5a98359ca7b).